terra-node-ci/README.md

```diff
@@ -0,0 +1,36 @@
+# terra-node-ci
+
+This dockerfile provides a docker image for terra projects to extend to use in CI it will copy code as well as run npm install by default.
+
+## API (of a sort)
+
+* Extends terra-node-base
+* On build: Copies . to /opt/module
+* On build: Runs Npm Install
```

Comment:
I'm all about documentation, but seems like it might be easier to maintaining by linking to the Dockerfile?? 🤷‍♀️

Reply:
eh, i left it, this doesn't change often